packages/common/ng-package.json

```diff
@@ -12,7 +12,7 @@
     { "input": "src", "glob": "*.scss", "output": "" },
     { "input": "src/lib", "glob": "**/*.theming.scss", "output": "lib" },
     { "input": "src/assets", "glob": "**/*", "output": "assets" },
-    { "input": "src/style/partial", "glob": "**/*.scss", "output": "style/partial" },
+    { "input": "src/style", "glob": "**/*.scss", "output": "src/style" },
```

Comment:
I add the `src` folder to limitate the difference between the raw importation with the symlink and the compiled/bundled importation